Mathlib/FieldTheory/IsAlgClosed/AlgebraicClosure.lean

```diff
@@ -19,8 +18,9 @@ In this file we construct the algebraic closure of a field
 - `AlgebraicClosure k` is an algebraic closure of `k` (in the same universe).
   It is constructed by taking the polynomial ring generated by indeterminates `x_f`
   corresponding to monic irreducible polynomials `f` with coefficients in `k`, and quotienting
-  out by a maximal ideal containing every `f(x_f)`, and then repeating this step countably
-  many times. See Exercise 1.13 in Atiyah--Macdonald.
+  out by a maximal ideal containing every `f(x_f)`. Usually this needs to be repeated countably
+  many times (see Exercise 1.13 in Atiyah--Macdonald), but using a theorem of Isaacs [Isaacs1980],
+  once is enough (see https://kconrad.math.uconn.edu/blurbs/galoistheory/algclosure.pdf).
```

Comment:
Maybe you could link Atiyah-Macdonald to references:

```suggestion
  many times (see Exercise 1.13 in (Atiyah--Macdonald)[atiyah-macdonald]), but using a theorem of
  Isaacs [Isaacs1980], once is enough
  (see https://kconrad.math.uconn.edu/blurbs/galoistheory/algclosure.pdf).
```

Reply:
I [changed it to a link](https://github.com/leanprover-community/mathlib4/pull/19853/commits/4e4c2d956c3d81e007fe118f67742ab853336379#diff-6722da230597017bb2621591c101e9fbd2ef0b6e34c8db47835c7d2a78594d0fL22-R22) but didn't repeat the names twice.